app/templates/foundry.html

```diff
@@ -33,7 +33,10 @@ <h1 class="page-title">ADI Foundry</h1>
                     <h4>Apply</h4>
                     <span class="bar gray"></span>
                 </div>
-                <h3>Applications have not opened yet. <a href="http://eepurl.com/jaugD">Sign up for the ADI newsletter</a> to be notified when applications open!</h3>
+                <h3>Applications are due September 23.</h3>
```

Comment:
Should we clarify that this is 2016? e.g. "Fall 2016 applications are due", or "are due Sept 23 2016"